Description: some tests use openssl client options that were dropped in version 1.1.0f-4,
 namely, -tls1, -tls1.1.
Forwarded: not-needed
Last-Update: 2017-08-16
Author: Jérémy Lal <kapouer@melix.org>
--- a/test/parallel/parallel.status
+++ b/test/parallel/parallel.status
@@ -6,6 +6,8 @@
 
 [true] # This section applies to all platforms
 
+test-tls-alert: SKIP
+
 [$system==win32]
 
 [$system==linux]
--- a/test/parallel/test-tls-session-cache.js
+++ b/test/parallel/test-tls-session-cache.js
@@ -97,7 +97,6 @@
   server.listen(0, function() {
     const args = [
       's_client',
-      '-tls1',
       '-connect', `localhost:${this.address().port}`,
       '-servername', 'ohgod',
       '-key', fixtures.path('agent.key'),
